wavefront/server/packages/flo_cloud/flo_cloud/aws/redshift.py

```
@@ -328,6 +329,52 @@ def execute_many(self, command: str, params_list: List[Dict[str, Any]]) -> int:
+            {
+                col: json.dumps(row.get(col))
+                if isinstance(row.get(col), (dict, list))
+                else row.get(col)
+                for col in columns
+            }
+            for row in data
+        ]
+
+        column_list = ', '.join(f'"{col}"' for col in columns)
+        placeholders = ', '.join(f':{col}' for col in columns)
+        command = f'INSERT INTO {table_name} ({column_list}) VALUES ({placeholders})'
```

Ruff (0.15.18)

[error] Possible SQL injection vector through string-based query construction

(S608)